src/js/common/schemaform/SaveInProgressErrorPage.jsx

```diff
@@ -0,0 +1,126 @@
+class SaveInProgressErrorPage extends React.Component {
+  getBackButton = (primary = false) => {
+    const buttonClass = primary ? 'usa-button-primary' : 'usa-button-outline';
+    const goBack = () => {
+      this.props.setFetchFormStatus(LOAD_STATUSES.notAttempted);
+      this.props.router.goBack();
+    };
+    return (
+      <ProgressButton
+          onButtonClick={goBack}
+          buttonClass={buttonClass}
+          buttonText="Back"
+          beforeText="«"/>
+    );
+  }
+
+  // Reload the form and try again.
+  reloadForm = () => {
+    // formConfig is put in this.props.routes[length - 1]
+    const formConfig = this.props.routes[this.props.routes.length - 1].formConfig;
+    this.props.fetchInProgressForm(formConfig.formId, formConfig.migrations);
+  }
+
+  render() {
+    const { loadedStatus } = this.props;
+    const { noAuth, notFound } = this.props.errorMessages || {};
+    let content;
+
+    switch (loadedStatus) {
+      case LOAD_STATUSES.noAuth:
+        content = (
+          <div>
+            <div className="usa-alert usa-alert-error no-background-image">You have been signed out. {noAuth}</div>
+            <div>
+              <div style={{ marginTop: '30px' }}>
+                {this.getBackButton()}
+                <SignInLink
+                    type="button"
+                    className="usa-button-primary"
+                    onLogin={this.reloadForm}
+                    isLoggedIn={this.props.isLoggedIn}
+                    loginUrl={this.props.loginUrl}
+                    onUpdateLoginUrl={this.props.updateLogInUrl}>Sign in</SignInLink>
+              </div>
+            </div>
+          </div>
+        );
+        break;
+      case LOAD_STATUSES.failure:
+        content = (
+          <div>
+            <div className="usa-alert usa-alert-error no-background-image">We're sorry, but something went wrong. Please try applying again in a few moments.</div>
+            <div style={{ marginTop: '30px' }}>
+              {this.getBackButton()}
+              <button className="usa-button-primary" onClick={this.reloadForm}>Resume previous application</button>
+            </div>
+          </div>
+        );
+        break;
+      case LOAD_STATUSES.notFound:
+        content = (
+          <div>
+            <div className="usa-alert usa-alert-error no-background-image">We're sorry, but something went wrong. We can't find your application. {notFound}</div>
+            <div style={{ marginTop: '30px' }}>
+              {this.getBackButton(true)}
+            </div>
+          </div>
+          // <button className="usa-button-primary" onClick={this.startOver}>Start over</button>
+        );
+        break;
+      default: // Shouldn't get here...
+        content = null;
+        break;
+    }
+
+    return (
+      <div>
+        {content}
+      </div>
+    );
+  }
+}
+
+SaveInProgressErrorPage.propTypes = {
+  formConfig: PropTypes.object.isRequired,
```

Comment:
Ah, yes...I thought passing it in the way I was in `helpers.js#createRoutes` would pass it as a prop. And...it doesn't...so.......yerp. 😞